unfetter-discover/unfetter-ui
# Issue 855

supports unfetter-discover/unfetter#855

requires PR https://github.com/unfetter-discover/unfetter-store/pull/146
## To Test
* pull this branch
* check travis
* start the ui in UAC run mode
* visit the Profile page
* select the framework a few times, ensure things are not broken
* leave the app, come back and see your last selection
* check the mongo user collection for your value
